prospector/autodetect.py

```diff
@@ -86,10 +87,8 @@ def autodetect_libraries(path: Union[str, Path]) -> set[str]:
 
     libraries: set[str] = set()
 
-    try:
+    with contextlib.suppress(RequirementsNotFound):
         libraries = find_from_requirements(path)
-    except RequirementsNotFound:
-        pass
```

Comment:
This is slower (I don't remember how much but noticeably) and is unusual imo we c/should skip this one.